addons/pos_stripe/models/pos_payment_method.py

```diff
@@ -65,7 +65,7 @@ def stripe_connection_token(self):
 
     def _stripe_calculate_amount(self, amount):
         currency = self.journal_id.currency_id or self.company_id.currency_id
-        return int(amount/currency.rounding)
+        return round(amount/currency.rounding)
```

Comment:
Wouldn't it be better to use currency.round(amount) ?

Reply:
No because they do not return the same values.
`round(1.18/currency.rounding)` would return `118`
`currency.round(1.18/currency.rounding)` would return `118.0`
They are used after to send to stripe and are not interpreted the same way

Reply:
Or use `int()` with the `currency.round()`  would work too, but not sure if it's necessary

Reply:
I added the `round` because there was a precision issue with the division that turned 1.18 into 117.99999999999 wich was interpreted as 117 by `int`